doc/config.example.yaml

```diff
@@ -1,35 +1,64 @@
 ################################################################################
-##                                  Sample keys                               ##
+##                                  Faucet Configuration                      ##
 ################################################################################
+web:
+  address: 127.0.0.1
+  port: 2766
 
-# When we deploy Faucet on the real TestNet, the keys used will not be well-known.
-keys:
-  # Genesis seed
-  - SDN7BBGE6Z6OQM3K4PACLTZUJ5QX4AY4QPDQ2JJ2JCFWCG2OIYYALIRY
-  # Well-known keypairs A to Z
-  - SAFRBTFVAB37EEJDIUGCDK5R3KSL3QDBO3SPS6GX752IILWB4NGQY7KJ
-  - SCX34MSZILQJT42XRSOEOZLI7NYFB67HTCCFWPIVEGRPIUY7GW2Q4CDJ
-  - SANGDLXBACY7J2LVUI5SONEOC4VTHHSYHXCDAKNSGZPGFN7O6N4AXEFE
-  - SDPVYLR53EAL2F4L3ACTBSZWVZU2WGAQFSABMMWC65M4GNXFQGMAQPAX
-  - SAY7YIWQZTMZYZYBH6H5IFFB6KBMC6H6UEAFZ2E2U23B6LLTYLTAP4BR
-  - SD2KCUN6M23OJV7DRIVE55VGM6GY6A7KXB6S2ANUVZKG2FSB4GZACBM4
-  - SBLCVET7XEHV6UCBGXNZO6PJEBS422T55OMVPPRCAW55AKJYTTNQHETJ
-  - SD4GL7SPYH5E3RZDIZQTY7BMORCGUARM72ZY7X62KXFY3IBNUAUA3RJD
-  - SAOR5XKXZXOMXN4FR4ALQGQ2WCXDFNATTJ2HYBE3TUH6KIH4SL2QQXQO
-  - SAKKQZIT5A7VG5BA646O2AR5HMOCYTGYIZS5L77N7JYU4G6C2CEQK3XC
-  - SDPZG67CSYW5VZH6NP7A35X3MHKYPCW6CWPXAAPEQCRZYOB5OVBA5G57
-  - SDSTVIZYOGEPF4QDN52PBARDPPKLGDNTVKN2EOM2K23SHHCHIRKQAC2H
-  - SAPHDQHRJCFY3YH5JR5RRXMQAY2AQNYEJTBFI7G46AOLXY5YBZMQ5JON
-  - SDPD4YQJDI2EGRBXNWCHG7GL74ITYUBQKFBD3CTHLIBQE2BNCEXAJBM4
-  - SDXA2R4JAPNFIL4RVDIGG5HIXNLMF6OKXM3FBX4QAMJ2QU3473UA2JGO
-  - SDW6PSPMER4P7SZ2BXDQPSFIXEIW6V26QCKLGUHVTFX6YCBOA35AWCAW
-  - SDYNPHQIDC4CMNMLKRKLHIKWDEQ6WVNESNNWMONIJJFODT6YXUQQ7S36
-  - SCCQRMDR63E6I5QPCYMQAXQ2NYULG562PKPLVSNTKQ6CEDGPXXWAYZQE
-  - SARGARQL5O7K7AGVPJ6W2MLUEKAKFCPCSP5P7O7ZV25GGBQE5VKAN2CT
-  - SDHNZA6CWQHHC2ZIZBAC4KZIJG2KL45RMGVPWQ226C6HQYX6N6ZQ7KDI
-  - SDAFJKB4IDJ5FRP5L2PL2TI6RZFVXBCNG36YW2YE3OM2UJTWOANQOID3
-  - SDD6TBCC7FTPJWMX5NJ5F2SVS5T3GA74KN7YWMEX3ALROS7PJ72QOOCM
-  - SBTL7LWIEMOVCPFFEKJLGYM6ZVE5GXCUX3ACI262YMVFWHJ2SQDABI4B
-  - SA63FZ3KRHA44PY56PTF44MX55MMJJHRR7TV4REEYAH6IGR5NSIA3FN4
-  - SAC6GZSDGDXF447OJK5MU6SAKVOP5WONYAIV2TIOFGXX2GPKUXZQ4ZAJ
-  - SAZWRKDD43DC5ETOW4L66VZK2U6RVAXHM7XJLAASYCMBQVQQ335AG7JA
+################################################################################
+##                  Transaction Generator Configuration                       ##
+################################################################################
+tx_generator:
+
+  # port number that can provide stats
+  stats_port: 9113
+
+  # How frequently we run our periodic task in seconds
+  send_interval: 1
+
+  # Between how many addresses we split a transaction by
+  split_count: 1
+
+  # Maximum number of utxo before merging instead of splitting
+  merge_threshold: 10
```

Comment:
The threshold seems to be a bit low. Should we increase?

Reply:
This is just an example file so IMO it does not really matter too much.